core/pages/Checkout.js

```diff
@@ -177,7 +177,12 @@ export default {
     onAfterPersonalDetails (receivedData, validationResult) {
       this.personalDetails = receivedData
       this.validationResults.personalDetails = validationResult
-      this.activateSection('shipping')
+
+      if (this.$store.state.cart.cartIsVirtual === true) {
```

Comment:
please access state only through getters -> isVirtualCart

Reply:
Okay

Reply:
Please review the latest commit - 90fa412